site/src/routes/+page.svelte

```diff
@@ -1,89 +1,99 @@
+                                        <td><a href="https://www.twitch.tv/{name}">{display_name}</a></td>
+                                        <td>{points}</td>
+                                        <td>
+                                            {#if marbles}
+                                                <svg style="width:512px;height:512px" viewBox="0 0 512 512">
+                                                    <path fill="currentColor"
+                                                          d="M168.06 172.176c-67.24 0-121.943 54.7-121.943 121.94S100.82 416.058 168.06 416.058c30.25 0 57.962-11.08 79.3-29.386-2.293-6.27-4.08-12.784-5.32-19.485-18.83 19.065-44.995 30.87-73.98 30.87-22.815 0-43.88-7.312-61-19.716 27.834 4.814 73.39 10.32 98.645.48 11.155-4.346 23.33-14.317 34.578-25.62-.184-2.73-.285-5.485-.285-8.263 0-16.365 3.25-31.983 9.123-46.256-21.226 20.275-49.207 43.894-71.65 51.906-26.097 9.316-76.967 3.676-96.025 1.09-8.796-13.23-14.635-28.602-16.593-45.182 17.335-3.294 53.808-10.013 74.972-12.38 28.105-3.143 60.837 10.417 84.705-4.75 14.41-9.16 25.482-38.016 30.31-52.555 6.01 9.1 10.62 19.205 13.543 30.015 4.347-5.198 9.116-10.03 14.258-14.44-17.077-46.75-61.993-80.21-114.58-80.21v-.002zm0 18c23.778 0 45.656 7.943 63.147 21.318-4.465 16.817-14.462 46.984-30.303 54.39-24.788 11.588-53.316-25.293-79.904-18.825-23.882 5.807-40.093 27.904-56.88 46.827.122-57.405 46.505-103.71 103.94-103.71zM361.94 241c-7.227 0-14.278.736-21.085 2.13 1.372 8.875 3.195 21.44 4.88 35.89 3.145 26.965 5.8 59.057 3.076 82.04-1.855 15.656-8.527 35.703-14.693 52.483-4.735 12.888-9.166 23.55-10.996 27.873 11.99 4.814 25.09 7.467 38.82 7.467 6.415 0 12.69-.58 18.777-1.686 4.906-16.186 14.49-50.886 13.226-71.197-1.047-16.833-19.873-31.586-16-48 5.268-22.31 39.43-47.627 55.56-58.52-18.62-17.66-43.798-28.48-71.56-28.48h-.004zm-38.495 7.334c-5.635 2.24-11.025 4.96-16.123 8.11.75 17.962 2.2 61.956-.424 88.443-2.483 25.083-12.448 66.07-15.12 76.814 4.704 4.3 9.807 8.172 15.245 11.556 1.973-4.694 5.937-14.32 10.2-25.92 5.975-16.26 12.268-36.212 13.713-48.397 2.313-19.514.002-51.422-3.08-77.835-1.49-12.778-3.12-24.257-4.41-32.77zm-33.71 21.77C270.158 288.994 258 315.514 258 344.94c0 22.48 7.092 43.264 19.166 60.246 4.123-17.823 10.2-45.726 11.818-62.073 1.874-18.918 1.378-51.442.752-73.01h-.002zm162.052 22.462c-6.606 14.014-16.406 36.63-18.34 51.434-2.54 19.446 3.155 53.058 6.41 69.813 16.2-18.325 26.026-42.423 26.026-68.872 0-19.114-5.135-37.004-14.096-52.374z"/>
+                                                </svg>
+                                            {/if}
+                                        </td>
+                                    </tr>
+                                {/each}
+                                </tbody>
+                            </table>
+                        </div>
+                    </div>
+                    <div class="card-footer">
+                        <footer>
```

Comment:
```suggestion
                    <footer class="card-footer">
```
ne peut pas suffire ?

Reply:
sûrement, je gardais juste la structure proposée j'adapte et au pire on rollback